Doc/library/functions.rst

```diff
@@ -989,8 +989,8 @@ are always available.  They are listed here in alphabetical order.
    already arranged into argument tuples, see :func:`itertools.starmap`\.
 
 
-.. function:: max(iterable, *[, key, default])
-              max(arg1, arg2, *args[, key])
+.. function:: max(iterable, *[, default=obj, key=func])
+              max(arg1, arg2, *args, *[, key=func])
```

Comment:
The documentation is better as it currently stands for this one. The `default` and `key` parameters are not easy to express in the signature. With your rewrite, `obj` and `func` would be undefined.